.yarnrc

```diff
@@ -0,0 +1 @@
+save-prefix false
```

Comment:
you mean like this https://github.com/meetup/web-platform-starter/blob/master/.yarnrc 😉